recipes/igraph-louvain/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{%set name = "louvain-igraph" %}
+{%set version = "0.5.3" %}
+{%set sha256 = "1bc4ea51eed5a6983ef3a1552c3261039502892eae9b4cb075a14ac59ecc8441" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/vtraag/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Could use PyPI here. Though if you stick with GitHub, please spell out `{{ name }}` instead of using Jinja as it doesn't really match the package name.